src/sentry/api/endpoints/project_rules_configuration.py

```diff
@@ -30,6 +30,7 @@ def get(self, request: Request, project) -> Response:
             "organizations:integrations-ticket-rules", project.organization
         )
         has_issue_severity_alerts = features.has("projects:first-event-severity-alerting", project)
+        has_high_priority_issue_alert = features.has("projects:high-priority-alerts", project)
```

Comment:
@snigdhas so we want a separate feature flag than `projects:first-event-severity-new-escalation`? cc @isabellaenriquez

Reply:
I think the alert should be separate from the escalation logic - if there's something wrong with the alert we don't need to stop detecting escalation or vice versa 

It _is_ one more thing to keep track of though..